runtime/doc/builtin.txt

```diff
@@ -1288,6 +1290,25 @@ blob2list({blob})					*blob2list()*
 <
 		Return type: list<any> or list<number>
 
+
+blob2str({blob})					*blob2str()*
+		Return a String by converting the UTF-8 encoded bytes in {blob}
+		into characters.
+
+							*E1515*
+		Returns an empty String if blob is empty or an invalid UTF-8
```

Comment:
This is the line I was referring to (and probably @chrisbra was as well); I didn't check the code.